### PR TITLE
Hack for knative monitoring

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Fixes #

## Proposed Changes

- hack to get the tests executred
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: test/e2e-tests.sh [hunk 1]
```diff
@@ -93,7 +93,11 @@ function knative_setup() {
   # TODO install head if !is_release_branch
   echo "Installing Knative Monitoring"
   kubectl create namespace istio-system
+  # Hack hack hack. Why is this namespace not created as part of monitoring release.
+  # https://github.com/knative/eventing/issues/3469
+  kubectl create ns knative-monitoring
```
Comment (mattmoor) [inline test/e2e-tests.sh:98]: Remove trailing whitespace:
```suggestion
  kubectl create ns knative-monitoring
```

**Comment by matzew** [comment]
see: https://github.com/vaikas/eventing/commit/00f1cc69124190c6e582e49a915f79cf94832c75

**Comment by matzew** [comment]
@aliok @slinkydeveloper mind looking here?

**Comment by slinkydeveloper** [comment]
Solved by https://github.com/knative/eventing-contrib/pull/1373

**Pull request closed without merging**